### DISCUSSION TITLE
White line over windows 11 taskbar

### DISCUSSION THREAD

**Discussion by currall** [valinet/ExplorerPatcher#883] [Q&A] [answered]
When I use the windows 11 taskbar there is a white line over all sections other than my pinned icons and action center. It only shows on the main monitor and only in windows 11 mode.

![rTJsybMVol](https://user-images.githubusercontent.com/29579820/154518958-40bb3538-5c30-4dfd-9d99-226b3c7b1ef3.png)

**Comment by valinet** [comment, marked as answer]
You posted this 3 days ago as well here: https://github.com/valinet/ExplorerPatcher/issues/871

Why make duplicate posts? I provided some feedback there, no reply...

**Reply by currall** [reply]
sorry, in your other post i thought you were asking me to repost in discussions, didnt see the answer
